addons/registry/2.7.1/install.sh

```diff
@@ -1,8 +1,6 @@
-
+# shellcheck disable=SC2148
```

Comment:
what does this disable?

Reply:
Not having the shebang at the beginning of the file.